src/app/static/src/app/components/header/LoggedOutHeader.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <header class="mb-5">
+        <nav class="navbar navbar-dark bg-secondary">
+            <div class="container-fluid">
+                <div class="navbar-header">
+                    {{title}}
```

Comment:
I suppose one day we might have different titles in French and English, but I'm assuming French for 'Naomi' is just Naomi with a French accent!

Reply:
I mean, I think Naomi makes as much sense in French as it does in English ;)